docs/index.md

```diff
@@ -76,15 +76,13 @@ const mapFile = new PMTiles(bostonMap._url);
 const buildingData = FileAttachment("data/buildings_data.csv").zip();
 ```
 
+<link rel="stylesheet" type="text/css" href="npm:maplibre-gl@4.0.2/dist/maplibre-gl.css">
```

Comment:
It looks like the stylesheet needs to be 
`<link rel="stylesheet" type="text/css" href="https://unpkg.com/maplibre-gl@4.0.2/dist/maplibre-gl.css"`, otherwise all attributions and navigation controls appear outside the canvas area.

Reply:
Did you run `npm install` and then restart the server? I forgot to mention that but it’s necessary to pick up the upgrade to 1.4.0. The issues you mention about the controls should go away if you upgrade.

Reply:
Ahhh ok, so using an abbreviated url needs the latest version. I've pushed up that change. Thanks!